media.css

```diff
@@ -0,0 +1,1400 @@
+    margin-left: 20px;
+  }
+
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    width: 41px;
+    height: 17px;
+    margin-right: 20px;
+  }
+
+  .top-content {
+    background-image: url(img/header-img-375.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 191px;
+  }
+
+  .top-content__block {
+    /* width: 355px;
+    height: 169px; */
+    padding-left: 20px;
+  }
+
+  .top-content__h1 {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 9px;
+  }
+
+  .link {
+    font-size: 12px;
+    line-height: 11px;
+    color: #fff;
+  }
+
+  .arrow {
+  	width: 40px;
+  	height: auto;
+  }
+
+  .hidden-768,
+  .hidden-375 {
+    display: none;
+  }
+
+  .catalog {
+    width: 281px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 0;
+  }
+
+  .catalog__title {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .catalog__items {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .catalog__item {
+    width: 281px;
+    height: 248px;
+  }
+
+  .catalog__item:not(:last-child)::after {
+    content: "";
+    display: none;
+  }
+
+  .catalog-item__image {
+    width: 165px;
+    height: 163px;
+    padding-left: 0;
+    display: block;
+    margin-bottom: 22px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .catalog-item__title {
+    font-size: 15px;
+    line-height: 14px;
+  }
+
+  .catalog-item__description {
+    grid-template-columns: 2fr 1fr;
+    padding-left: 15px;
+    padding-right: 0;
+  }
+
+  .catalog-item__price {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .catalog-item__grey,
+  .catalog-item__black,
+  .catalog-item__red,
+  .catalog-item__orange,
+  .catalog-item__brown,
+  .catalog-item__cherry,
+  .catalog-item__blue {
+    height: 15px;
+    width: 15px;
+  }
+
+  .video-section {
+    height: 505px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: linear-gradient(2.6deg, #000 91.5%, #fff 92%);
+    margin-bottom: 0;
+  }
+
+  .video-section__image {
+    display: flex;
+    justify-content: center;
+    order: 1;
+  }
+
+  .video {
+    display: none;
+  }
+
+  .video-adaptive {
+    display: flex;
+    justify-content: center;
+    width: 86%;
+    margin: 0 auto;
+  }
+
+  .video-section__content {
+    display: flex;
+    flex-direction: column;
+    order: -1;
+    width: 86%;
+    align-items: center;
+  }
+
+  .video-content__h3 {
+    font-size: 20px;
+    line-height: 25px;
+    margin-bottom: 15px;
+    color: #fff;
+    text-transform: uppercase;
+    width: 100%;
+    text-align: center;
+  }
+
+  .video-content__description {
+    width: 100%;
+    font-size: 14px;
+    line-height: 17px;
+    margin-bottom: 41px;
+    color: #fff;
+    text-align: center;
+  }
+
+  .catalog-item__title {
+  	padding-left: 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)